### DISCUSSION TITLE
[Not necessary] Sending n number of snaps from a single snap

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [rodit/SnapMod#117] [Ideas & Feature Suggestions]
Like, as you know the number of snaps we send, the more snapscore we make. So better if we send a snap and it will go repeated n number of times like 1000 or something and we can get faster snapscore by sending snaps to randomly added people and dead accounts.

I already tried this and have over a million snapscore but the thing is now if it would be possible to add a spam option and send a single snap repeated times to achieve many snapscore at once.

Thought it is quite hilarious to listen and not at all required feature, already you have added many great features and I've suggested features 2 times in the discussion tab that has already been added so many many thanks for that. But anyways, it's a good thing for you to try. Btw Great Project❤️

**Comment by rodit** [comment]
I don't think I'll implement this tbh. Doesn't really fit the purpose of the project.

**Reply by RevealedSoulEven** [reply]
Yeaaa that's what I said too. But it will help to gain a lot snapscore and that will make the project super awesome

**Comment by Magna-Graecia** [comment]
The thing is that you have to take into account UI clutter. While having more and more options is great, it will become annoying and cumbersome to navigate context menus and even the SnapMod settings app that's already jampacked with tons of options if it gets filled with unnecessary ones, so this should be avoided in my opinion.

**Reply by RevealedSoulEven** [reply]
Yes you are right. What about creating a new project like snapbomber?
I guess it existed in past to spam on Snapchat

**Reply by RevealedSoulEven** [reply]
After investigation, I came to know that snapbomber was for windows only. So why not try something for android? Btw it's not at all required

**Reply by rodit** [reply]
I did like the idea of making some sort of scriptable interface for Snapchat to control it so this maybe something I work on in the future. It is quite a big undertaking though so it probably won't happen anytime soon.

**Reply by GoatBan** [reply]
Just use repetitouch & make your own script, it's easy to setup & I used to boost some account up to 2m in 2-3 days

**Reply by RevealedSoulEven** [reply]
@GoatBan But for that you need to keep your device open and busy I guess

**Reply by GoatBan** [reply]
> @GoatBan But for that you need to keep your device open and busy I guess

Yup, but from what i've since, all the boosting service are using something similar (except for justxn who was using a website)

**Reply by RevealedSoulEven** [reply]
Aahhhh yea I got it
